panel/models/vtk/vtkjs.ts

```diff
@@ -8,7 +8,8 @@ export class VTKJSPlotView extends AbstractVTKView {
 
   override connect_signals(): void {
     super.connect_signals()
-    this.connect(this.model.properties.data.change, () => {
+    const {data} = this.model.properties
+    this.on_change(data, () => {
```

Comment:
Why not keep this as one line?

Reply:
Consistency with all other changes I made.

Reply:
I think you are a bit to consistent then. I don't see any problem with doing what you wrote in the original post: `this.on_change(this.model.properties.some, () => ...)`

But I don't have any strong opinions about this.